code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,322 @@
+/*
+Destabilized extracts:
+	Throwable extracts that apply an effect on hit.
+*/
+
+/obj/item/slimecross/destabilized
+	name = "destabilized extract"
+	desc = "It seems ready to burst at any moment."
+	effect = "destabilized"
+	icon_state = "destabilized"
+	amountToCreate = 5
+
+/obj/item/slimecross/destabilized/throw_impact(atom/A, var/datum/thrownthing/T)
+	if(!..()) //For those tactical catches
+		if(hitEffect(A, T.thrower))
+			playsound(get_turf(A), 'sound/misc/splort.ogg', get_volume_by_throwforce_and_or_w_class(), 1, -1)
+			qdel(src)
+
+/obj/item/slimecross/destabilized/proc/hitEffect(atom/A, mob/thrower)
+	return TRUE //But nothing happens...
+
+/obj/item/slimecross/destabilized/grey
+	colour = "grey"
+
+/obj/item/slimecross/destabilized/grey/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/mob/living/simple_animal/slime/S = new (get_turf(A))
+	S.rabid = TRUE
+	S.Friends += thrower
+	if(isliving(A))
+		var/mob/living/L = A
+		S.Target = L
+		if(S.CanFeedon(L))
+			S.Feedon(L)
+
+/obj/item/slimecross/destabilized/orange
+	colour = "orange"
+
+/obj/item/slimecross/destabilized/orange/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		if(L.fire_stacks < 1)
+			L.fire_stacks = 1
+		L.IgniteMob()
+	else
+		new /obj/effect/hotspot(get_turf(A))
+
+/obj/item/slimecross/destabilized/purple
+	colour = "purple"
+
+/obj/item/slimecross/destabilized/purple/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.reagents.add_reagent(/datum/reagent/medicine/regen_jelly, 50)
+	else
+		var/datum/reagents/R = new/datum/reagents(50)
+		R.add_reagent(/datum/reagent/medicine/regen_jelly, 50)
+		var/datum/effect_system/smoke_spread/chem/smoke = new
+		smoke.set_up(R, 1, get_turf(A))
+		smoke.start()
+
+/obj/item/slimecross/destabilized/blue
+	colour = "blue"
+
+/obj/item/slimecross/destabilized/blue/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/datum/reagents/R = new/datum/reagents(50)
+	R.add_reagent(/datum/reagent/water, 50)
+	R.reaction(get_turf(A), TOUCH, 50)
+	qdel(R) //Reaction doesnt use up the reagents
+	if(iscarbon(A))
+		var/mob/living/carbon/C = A
+		C.slip(100, src, NONE, 20, FALSE)
+
+/obj/item/slimecross/destabilized/metal
+	colour = "metal"
+
+/obj/item/slimecross/destabilized/metal/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
```

Comment:
does this work on megafauna? this could be a really easy way to get one to the station

Reply:
it's also basically a one-shot against most NPCs, which I guess is fine if you make it exclude bosses (like the paper wizard)

Reply:
I'm not sure how this is a "one shot" to anything, as it does no damage. This doesn't let you do anything you couldn't already do with a bluespace bodybag, with the added downside that, unlike the bag, you can't put it in your backpack.